charts/quickwit/templates/indexer-statefulset.yaml

```diff
@@ -55,6 +55,15 @@ spec:
             - name: "{{ $key }}"
               value: "{{ $value }}"
             {{- end }}
+          {{- if or (.Values.environmentFrom) (.Values.indexer.extraEnvFrom) }}
+          envFrom:
+          {{- with .Values.environmentFrom }}
+            {{- toYaml . | nindent 12 }}
+          {{- end }}
+          {{- with .Values.indexer.extraEnvFrom }}
```

Comment:
See this comment: https://github.com/quickwit-oss/helm-charts/pull/85#issuecomment-2176463986